build.proj

```diff
@@ -71,6 +71,18 @@
   <Import Project="dir.targets" />
   <Import Project="dir.traversal.targets" />
   <Import Project="$(ToolsDir)clean.targets" />
+  
+  <Target Name="CollectTemplateFiles" BeforeTargets="UpdateDependencies" >
+    <ItemGroup>
+      <ProjectJsonTemplateFiles Include="$(MSBuildThisFileDirectory)/tools-local/**/project.json.template" />
+      <ProjectJsonTemplateFiles Include="$(MSBuildThisFileDirectory)/src/**/project.json.template" />
+      <ProjectJsonFiles Remove="*.json" />
```

Comment:
They are getting added in dir.props but you're right we still use the project.jsons at least for tests, will add them back